tools/wt_verify.py

```diff
@@ -131,28 +234,27 @@ def main():
 
     try:
         command = construct_command(args)
-        output = execute_command(command)
+        wt_output_file = execute_command(command, "wt_output_file.txt")
```

Comment:
could we have this as a global var as a constant, 
`WT_OUTPUT = "wt_output_file.txt"`. 
Instead of passing and returning in the function

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10164/commits/10265d29a0c1a8d7f101c1207e3dcb3620b3a1ef